packages/api-browser-demo/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ssf-desktop-api-browser-demo",
+  "version": "0.0.1",
+  "description": "",
+  "main": "",
+  "scripts": {
+    "serve": "live-server --port=5000 src/",
+    "start": "npm run serve",
```

Comment:
Could probably remove `serve` and just have `start` as `live-server --port=5000 src/`

Not sure if it's worth specifying the port - could remove?